contracts/contracts/strategies/balancer/BalancerMetaPoolStrategy.sol

```diff
@@ -123,6 +123,8 @@ contract BalancerMetaPoolStrategy is BaseAuraStrategy {
             _strategyAssets.length == _strategyAmounts.length,
             "Array length missmatch"
         );
+        // store poolAssets storage variable into memory for gas optimizations
+        address[] memory poolAssets = poolAssets;
```

Comment:
🔴 I don't thinking we should shadow and use the same variable name. The local one should get an "_"

Reply:
Great point, fixed as suggested